deltachat-ffi/deltachat.h

```diff
@@ -1154,7 +1154,13 @@ uint32_t dc_send_videochat_invitation (dc_context_t* context, uint32_t chat_id);
  * @memberof dc_context_t
  * @param context The context object.
  * @param msg_id The ID of the message with the webxdc instance.
- * @param json program-readable data, the actual payload
+ * @param json program-readable data, this is create in JS land as:
+ *     - `payload`: any JS object or primitive.
+ *     - `info`: optional informational message. will be shown in chat an may be added as system notification.
+ *     - `notify`: optional array of user `addr` that should be notified eg. by a sound.
+ *       note that still all users get the update payload and the `info` message shown in a chat.
+ *     - `document`: optional document name. shown eg. in title bar.
+ *     - `summary`: optional summary. shown beside app icon.
  * @param descr The user-visible description of JSON data,
```

Comment:
offtopic: would be nice if we deprecate this, internally just ignore it

Reply:
yip, i think, we already agreed on that, but that is offtopic